docs/guides/identity-providers/okta.md

```diff
@@ -15,27 +15,38 @@ infra providers add okta \
 ## Finding required values
 
 1. Login to the Okta dashboard.
-2. Under the left menu click **Applications > Applications**.  
-   Click **Create App Integration**.  
-   Select **OIDC – OpenID Connect** and **Web Application**.  
-   Click **Next**.
+2. Navigate to **Applications > Applications**
 
 ![Create Application](../../images/connect-users-okta-okta1.png)
 
-3. For **App integration name** write **Infra**.  
-   In **General Settings** > **Grant type** select **Authorization Code** and **Refresh Token**.  
-   For **Sign-in redirect URIs** write `http://localhost:8301`. For **Assignments** select the groups which will have access through Infra.  
-   Click **Save**.
+3. Create an Okta App:
+   a. Click **Create App Integration**.
+   b. Select **OIDC – OpenID Connect** and **Web Application**.
+   c. Click **Next**.
 
 ![App Type](../../images/connect-users-okta-okta2.png)
 
-4. On the **General** tab, **note** the **Client ID**, **Client Secret**, and **Okta domain** for adding your Okta information to Infra later.
+4. Configure your new Okta App:
+  - For **App integration name** write **Infra**.
+  - Under **General Settings** > **Grant type** select **Authorization Code** and **Refresh Token**.
+  - For **Sign-in redirect URIs** write:
+    - `http://localhost:8301` (for CLI login)
```

Comment:
This is re-worked to add UI support.

Note: we should consider moving all OIDC flows to the UI once `--enable-ui` is `true` by default.

For two reasons:
1.  Users only have to add a single redirect URL
2. Users will be able to log in environments without a browser (e.g. ssh, a docker container)